### ISSUE TITLE
Formatting wrong in env example

### ISSUE THREAD

**Issue by jmandel** [MicrosoftDocs/vsts-docs#5037]
Regarding the following block:

```
- script: echo $MYSECRET
  env:
    MySecret: $(Foo)
# equivalent using task syntax - note that this is NOT a task input

task: Bash@3
inputs:
type: inline
script: echo $MYSECRET
env:
MySecret: $(Foo)
```

The last line should be indented; the `inputs:` line is blank and probably shouldn't be present; the `#`-prefixed comment line is cryptic, and seems to be saying that these two snippets should be separated out into two distinct code blocks. Overall this is very hard to follow and appears to include mistakes.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 450ac349-5c33-6cdf-1ecd-6eb989a00334
* Version Independent ID: a7119c1a-59b2-3081-a241-e93aa0d6b4b9
* Content: [Bash task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/bash?view=azure-devops)
* Content Source: [docs/pipelines/tasks/utility/bash.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/tasks/utility/bash.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @vtbassmatt
* Microsoft Alias: **macoope**

**Comment by msebolt** [comment]
@jmandel  Sorry for the delay and thank you for the feedback.

The sample has been updated, changes pending review.
